bedrock/firefox/templates/firefox/tiles.html

```diff
@@ -31,9 +31,6 @@
   {{ fxfamilynav('desktop', '', 'dark') }}
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
This page has confusing CSS inheritance. Might be better to switch this template from `site_css` to `page_css`?